ee/desktop/runner/runner_linux.go

```diff
@@ -54,5 +56,31 @@ func runAsUser(uid string, cmd *exec.Cmd) error {
 		},
 	}
 
+	// Get the user's session so we can get their display -- this is needed only for handling notifications
+	// at the moment, so we ignore any errors
+	ctx := context.Background()
+	sessionOutput, _ := exec.CommandContext(ctx, "loginctl", "show-user", uid, "--value", "--property=Sessions").Output()
```

Comment:
Should this be a new context? 

Regardless, it should have a timeout.

Reply:
error handling?

Reply:
Hrm. This whole block is about trying to set some things into `cmd.Env` right?

I'd suggest we make it a helper func with a sig like `getDisplayInfo(uid int) map[string]string`

Reply:
This whole bit is still a bit TBD as I figure out the wayland issue -- it needs some tidying up! I can pull it out into a helper func once it's finalized.

Reply:
I've updated this to make a separate function and add a timeout, and additionally passed in the logger so that we can log errors. I don't think it makes sense to stop the desktop process from launching if anything here fails, so I think that should be all we need re: error handling too.